org.freeplane.App.metainfo.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It should be upstreamed.

Reply:
https://github.com/freeplane/freeplane/pull/157